packages/lesswrong/components/editor/LocalStorageCheck.tsx

```diff
@@ -8,6 +8,16 @@ const styles = (theme: ThemeType): JssStyles => ({
     ...theme.typography.commentStyle,
     color: theme.palette.text.normal,
     paddingBottom: 12,
+    
+    border: theme.palette.border.normal,
+    padding: 8,
+    borderRadius: 8,
+    backgroundColor: theme.palette.panelBackground.restoreSavedContentNotice,
+    margin: 10,
```

Comment:
And take away the left margin to make it left aligned along the vertical ruler created by the linkpost field and text.